### PR TITLE
feat: readme monitoring event

### PR DESCRIPTION
**Pull request by RickaPrincy**

### CODE REVIEW

File: src/main/java/app/bpartners/geojobs/service/event/readme/ReadmeLogCreatedService.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  public void accept(ReadmeLogCreated readmeLogCreated) {
+    var readmeMonitorConf = readmeLogCreated.getReadmeMonitorConf();
+    var readmeLog = readmeLogCreated.getReadmeLog();
+
+    if (readmeMonitorConf.isDevelopment() != readmeLog.development()) {
+      throw new BadRequestException(
+          "readmeLog.development should be " + readmeMonitorConf.isDevelopment());
+    }
+
+    String requestBody = objectMapper.writeValueAsString(List.of(readmeLog));
```
Comment (nvagno) [inline src/main/java/app/bpartners/geojobs/service/event/readme/ReadmeLogCreatedService.java:63]: Put it into another method, then handle exception with try/catch and remove `@SneakyThrows`

**Review by nvagno — CHANGES_REQUESTED** [review]

File: src/main/java/app/bpartners/geojobs/endpoint/rest/readme/monitor/factory/ReadmeEntryFactory.java [hunk 2]
```diff
@@ -26,7 +25,7 @@ public ReadmeEntry createReadmeEntry(
         .request(
             ReadmeEntryRequest.builder()
                 .method(request.getMethod())
-                .httpVersion(HTTP_2.toString())
+                .httpVersion(ReadmeLogCreatedService.getClientVersion().toString())
```
Comment (hei-ryan) [inline src/main/java/app/bpartners/geojobs/endpoint/rest/readme/monitor/factory/ReadmeEntryFactory.java]: Import directly getClientVersion()

File: src/main/java/app/bpartners/geojobs/service/event/readme/ReadmeLogCreatedService.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    HttpRequest httpRequest =
+        HttpRequest.newBuilder()
+            .uri(URI.create(readmeMonitorConf.getUrl()))
+            .header("Content-Type", README_API_MIME_TYPE)
+            .header("Authorization", getBasicAuthValue(readmeMonitorConf))
+            .POST(BodyPublishers.ofString(requestBody))
+            .build();
+    HttpResponse<String> readmeResponse = httpClient.send(httpRequest, BodyHandlers.ofString());
+
+    log.info("readme.monitor.requestBody: {}", requestBody);
```
Comment (hei-ryan) [inline src/main/java/app/bpartners/geojobs/service/event/readme/ReadmeLogCreatedService.java:73]: Are those logs really needed for monitoring console log ?

**Review by hei-ryan — CHANGES_REQUESTED** [review]

File: src/main/java/app/bpartners/geojobs/service/event/readme/ReadmeLogCreatedService.java (hunk 3 above)
Comment (RickaPrincy) [reply src/main/java/app/bpartners/geojobs/service/event/readme/ReadmeLogCreatedService.java:73]: Yes, it's only for debugging while I'm testing. Once it's done, I will remove it.

**Pull request merged**